instrumentation/okhttp/okhttp-3.0/library/src/main/java/io/opentelemetry/instrumentation/okhttp/v3_0/TracingCallFactory.java

```diff
@@ -24,22 +23,19 @@
   private static final Cache<Request, Context> contextsByRequest =
       Cache.newBuilder().setWeakKeys().build();
 
-  @Nullable private static MethodHandle timeoutMethodHandle;
-  @Nullable private static MethodHandle cloneMethodHandle;
+  @Nullable private static Method timeoutMethod;
```

Comment:
can you add a comment here about not being able to use method handles (so no one changes it back), at least until we get some build tooling in place to prevent similar to otel-java repo?

Reply:
done